spyder/plugins/updatemanager/scripts/install.bat

```diff
@@ -24,20 +26,49 @@ echo.
 
 call :wait_for_spyder_quit
 
-IF not "%conda%"=="" IF not "%prefix%"=="" (
+IF exist "%conda%" IF exist "%prefix%" IF exist "%base_file%" IF exist "%runtime_file%" (
     call :update_subroutine
+    set /P =Press return to exit and launch Spyder...
     call :launch_spyder
     goto exit
 )
 
-IF not "%install_file%"=="" (
+IF exist "%install_file%" (
     call :install_subroutine
     goto exit
 )
 
 :exit
 exit %ERRORLEVEL%
 
+:wait_for_spyder_quit
```

Comment:
How can we check that these changes to the update script are working as expected?

Reply:
I've checked the update process for all three platforms using my local mac and Windows and Linux VMs. I did this by installing 6.0.0b3 using the installer, then patching the runtime environment with the changes in this PR and also spoofing the Spyder version to 6.0.0b2 and also changing the Github url to point to my fork. My fork has the runtime and base environment lock files in the 6.0.0b3 release.

Is this what you meant?

Reply:
Yeah, it is. Thanks for doing the extra step of manually checking that the update process is working as expected with the changes you've done here.